### ISSUE TITLE
env vars being cached

### ISSUE THREAD

**Issue by elie222** [zodern/meteor-up#1069]
### Mup version (`mup --version`):
1.4.5, 1.4.4

### Mup config

```js
{
  "servers": {
    "server0": {
      "host": "1.2.3.4",
      "username": "xxx",
      "pem": "~/.ssh/pem"
    }
  },
  "app": {
    "name": "my-app",
    "path": "../../..",
    "servers": {
      "server0": {}
    },
    "buildOptions": {
      "serverOnly": true
    },
    "env": {
      "HTTP_FORWARDED_COUNT": 1,
      "ROOT_URL": "https://subdomain.host.com",
      "MONGO_URL": "xxx",
      "MONGO_OPLOG_URL": "xxx",
      "MAIL_URL": "xxx",
      "APOLLO_ENGINE_API_KEY": "xxx",
      "SENDGRID_API_KEY": "xxx",
      "GENERAL_WORKER": "false",
      "DRAFT_WORKER": "false",
      "AUCTION_WORKER": "false",
      "MONITOR_USERS": "false",
      "DRAFT_COMPLETER": "false",
      "TELEGRAM_BOT_WORKER": "false",
      "PUSH_NOTIFIER_WORKER": "false"
    },
    "docker": {
      "image": "abernix/meteord:node-8.11.2-base",
      "args": [
        "--memory 1200M"
      ]
    },
    "deployCheckWaitTime": 90,
    "enableUploadProgressBar": true,
    "type": "meteor"
  },
  "hooks": {
    "post.reconfig": {
      "remoteCommand": "sh ~/rebuild-restart-mup-draft-app.sh"
    }
  }
}
```

### Output of command

The deploy works fine. The issue is that the env_var `CDN_URL` is not unset. As can be seen, it's not in the config above. Changing it does work, but I am unable to remove it.

Having created a new server, and done `mup setup` again on it. Things worked fine. I believe the issue may be that if you set the env var in setup, it cannot be unset. Running `mup setup` again does not help.

**Comment by zodern** [comment]
This will be fixed in Mup 1.5: https://github.com/zodern/meteor-up/pull/976/commits/289837df6af097de95ace1fc4f15ae284fdb7949
`mup deploy` is able to remove the environment variables in older versions of Mup.

To clarify, `mup setup` does not modify the app's environment variables. `mup reconfig` should be used instead. Since the difference of the two commands is confusing, I want to merge it into `mup reconfig`: https://github.com/zodern/meteor-up/issues/915

**Comment by elie222** [comment]
Ok. I wasn’t trying mup setup as a first avenue. I think it’s clear enough
it should be used on setup. Although I think there’s a message that says
env set up when your run mup setup.

**Comment by elie222** [comment]
Yup. What you wrote is correct. Running `mup deploy` did fix it. Thanks. Annoying bug :(

**Comment by x5engine** [comment]
please close this if fixed thanks

**Comment by zodern** [comment]
It is fixed in the 1.5 beta.
